apps/hash-graph/lib/graph/src/shared/subgraph.rs

```diff
@@ -27,24 +27,24 @@ pub struct Subgraph {
     pub vertices: Vertices,
     pub edges: Edges,
     pub depths: GraphResolveDepths,
-    pub time_projection: UnresolvedTimeProjection,
-    pub resolved_time_projection: TimeProjection,
+    pub temporal_axes: UnresolvedTemporalAxes,
+    pub resolved_temporal_axes: TemporalAxes,
```

Comment:
The fact that these are swapped seems quite strange (as in the variable name vs the type)

Reply:
Fixed in [Use same subgraph representation in Rust](https://github.com/hashintel/hash/pull/1854/commits/6c9a54f48ac7f65b8b521a313e3c95ab60e5372b)